src/main/java/io/sqooba/traildb/TrailDBEvent.java

```diff
@@ -17,7 +18,7 @@
 
     private long timestamp;
     private long numItems;
-    private long[] items;
+    private long items;
```

Comment:
Does that mean that now you only pass the pointer to the items value ?

Reply:
Yes. To the encoded items (64-bit integer).